release-notes/swan-lake-2201.1.0-release-note.md

```diff
@@ -112,6 +112,10 @@ To view bug fixes, see the GitHub milestone for Swan Lake 2201.1.0 of the reposi
 
 #### New Features
 
+##### AsyncAPI Tool
+
+- Ballerina AsyncAPI tooling will make it easy for you to start the development of an event API documented in an AsyncAPI contract in Ballerina by generating Ballerina service and listener skeletons. Ballerina Swan Lake supports the AsyncAPI Specification version 2.x. For more information, see [ballerina-asyncapi-support](http://ballerina.io/learn/ballerina-asyncapi-support) and [cli-documentation](http://ballerina.io/learn/cli-documentation/asyncapi/#asyncapi-to-ballerina).
```

Comment:
```suggestion
- Ballerina AsyncAPI tooling will make it easy for you to start the development of an event API documented in an AsyncAPI contract in Ballerina by generating a Ballerina service and listener skeletons. Ballerina Swan Lake supports the AsyncAPI Specification version 2.x. For more information, see [Ballerina AsyncAPI support](http://ballerina.io/learn/ballerina-asyncapi-support) and [AsyncAPI CLI documentation](http://ballerina.io/learn/cli-documentation/asyncapi/#asyncapi-to-ballerina).
```

Reply:
Thanks a lot for the suggestion. Resolved